vcd/datasource_vcd_independent_disk.go

```diff
@@ -0,0 +1,69 @@
+package vcd
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func datasourceVcIndependentDisk() *schema.Resource {
+	return &schema.Resource{
+		Read: resourceVcdIndependentDiskRead,
+		Schema: map[string]*schema.Schema{
+			"vdc": {
```

Comment:
Where is the `org`, why it's not here?

Reply:
Because we don't use or need it. Disk depends to vdc, not Org. https://code.vmware.com/apis/287/vcloud/doc/doc/operations/POST-CreateDisk.html

Reply:
How do you get the VDC if you don't know the Org?

Reply:
org deprecation removed

Reply:
Thanks!